src/Sylius/Bundle/ProductBundle/Doctrine/ORM/ProductVariantRepository.php

```diff
@@ -85,6 +85,17 @@ public function findOneByCodeAndProductCode(string $code, string $productCode):
         ;
     }
 
+    public function findOneByCode(string $code): ?ProductVariantInterface
```

Comment:
do you need this function? you can use simple `findOneBy(["code" => $productVariantCode])`